EIPS/eip-7231.md

```diff
@@ -0,0 +1,175 @@
+---
+eip: 7231
+title: Identity aggregated NFT
+description: The aggregation of web2 & web3 identities to NFTs, authorized by individuals, gives attributes of ownerships, relationships, experiences.
+author: Chloe Gu <chloe@carv.io>, Navid X. (@navid), Victor Yu <victor@carv.io>, Archer H.
```

Comment:
You'll want to add your GitHub username (`xuxinlai2002`) here or else you won't be able to edit your own EIP.

Reply:
modify @navid -> @xuxinlai2002